packages/config/config/devices/0x027a/zse11.json

```diff
@@ -2,13 +2,25 @@
 {
 	"manufacturer": "Zooz",
 	"manufacturerId": "0x027a",
-	"label": "ZSE11",
+	"label": [
+		{
+			"$if": "firmwareVersion >= 2.0",
+			"value": "ZSE11 800LR"
+		},
+		"ZSE11"
+	],
```

Comment:
We've actually removed this distinction including any mention of the Z-Wave series in labels after Zooz requested it.
So let's keep it as it was.